src/CSS/App.css

```diff
@@ -4,13 +4,7 @@ body {
   display: flex;
   min-height: 100vh;
 }
-.App h1 {
-  text-align: center;
-  font-size: 2em;
-  color: darkblue;
-  
 
-}
 
 .App ul {
   list-style: none;
```

Comment:
I think styles from line 30 to 64 were related to boilerplate react vite app, you don't need them now.